aea/crypto/ledger_apis.py

```diff
@@ -69,33 +75,66 @@ def token_balance(self, identifier: str, address: str) -> int:
+        api = self.apis[identifier]
+        if identifier == FETCHAI:
+            try:
+                balance = api.tokens.balance(address)
+            except Exception:
+                logger.warning("An error occurred while attempting to get the current balance.")
+                balance = 0
+        elif identifier == ETHEREUM:
+            try:
+                balance = api.eth.getBalance(address)
+            except Exception:
+                logger.warning("An error occurred while attempting to get the current balance.")
+                balance = 0
+        else:
             balance = 0
         return balance
 
-    def transfer(self, identifier: str, entity: Entity, destination_address: str, amount: int, tx_fee: int) -> Optional[str]:
+    def transfer(self, identifier: str, crypto_object: Crypto, destination_address: str, amount: int, tx_fee: int) -> Optional[str]:
         """
         Transfer from self to destination.
 
+        :param identifier: the crypto code
+        :param crypto_object: the crypto object that contains the fucntions for signing transactions.
         :param destination_address: the address of the receive
         :param amount: the amount
         :param tx_fee: the tx fee
 
         :return: tx digest if successful, otherwise None
         """
         assert identifier in self.apis.keys(), "Unsupported ledger identifier."
-        try:
-            api = self.apis[identifier]
-            logger.info("Waiting for the validation of the transaction ...")
-            tx_digest = api.tokens.transfer(entity, destination_address, amount, tx_fee)
-            api.sync(tx_digest)
-            logger.info("Transaction validated ...")
-        except Exception:
-            logger.warning("An error occurred while attempting the transfer.")
+        api = self.apis[identifier]
+        logger.info("Waiting for the validation of the transaction ...")
+        if identifier == FETCHAI:
+            try:
+                tx_digest = api.tokens.transfer(crypto_object.entity, destination_address, amount, tx_fee)
+                api.sync(tx_digest)
+                logger.info("Transaction validated ...")
+            except Exception:
+                logger.warning("An error occurred while attempting the transfer.")
+                tx_digest = None
+        elif identifier == ETHEREUM:
+            try:
+                transaction = {
+                    'nonce': 0x00,
+                    'chainId': 3,
+                    'to': api.toChecksumAddress(destination_address),
+                    'value': amount,
+                    'gas': tx_fee + api.eth.gasPrice,
+                    'gasPrice': api.eth.gasPrice
+                }
+
+                signature = api.eth.account.sign_transaction(transaction_dict=transaction,
+                                                             private_key=crypto_object.entity.privateKey)
+
+                logger.info("Signature: {}".format(signature))
```

Comment:
Info should not print this. The user does not need to know this. You could print as part of `debug`. Please add the same `info` messages as above in fetchai case.